components/omega/doc/design/OmegaV1GoverningEqns.md

```diff
@@ -0,0 +1,783 @@
+1. How do I justify changing from $\nabla_z$ to $\nabla_r$ for terms other than the pressure gradient? Particularly $\nabla K$?
+
+### General Vertical Coordinate
+
+The vertical layer interfaces $(z_k^{bot}, z_k^{top})$ (or equivalently $(p_k^{bot}, p_k^{top})$) can vary as a function of $(x,y,t)$. Thus, these equations describe a general vertical coordinate, and these interface surfaces may be chosen arbitrarily by the user. See [Adcroft and Hallberg 2006](https://www.sciencedirect.com/science/article/pii/S1463500305000090) Section 2 and [Griffies et al](http://sciencedirect.com/science/article/pii/S1463500300000147) Section 2.  It is convenient to introduce a new variable, the coordinate $r(x,y,z,t)$, where $r$ is constant at the middle of each layer. To be specific, we could design $r$ to be the layer index $k$ at the mid-depth of the layer. That is, define the mid-depth as
+
+$$
+z_k^{mid}(x,y,t) = \frac{z_k^{bot}(x,y,t) +  z_k^{top}(x,y,t)}{2}
+$$ (def-z)
+and generate the function $r$ such that
+
+$$
+r(x,y,z_k^{mid}(x,y,t),t) = k
+$$ (def-r)
+and interpolate linearly in the vertical between mid-layers. When we write the layered form of the equations, we must take into account of the tilted layers using the chain rule.
+
+We now rewrite derivatives in order to convert from horizontal coordinates to tilted coordinates. Let $(x,y)$ be the original horizontal coordinates, which are perpendicular to $z$, and the horizontal gradient be written as $\nabla_z=(\partial/\partial x, \partial/\partial y)$, as above. Now define a tilted coordinate system using the layers defined by $r$, where the within-layer horizontal coordinates are $(x',y')$ and the along-layer gradient is written as $\nabla_r=(\partial/\partial x', \partial/\partial y')$. We construct $r$ to be monotonic in $z$, so we can invert it as $z(x',y',r,t)$. Now horizontal derivatives along the tilted direction $x'$ for any field $\varphi(x,y,z,t)$ can be expanded using the chain rule as
+
+$$
+\frac{\partial }{\partial x'} \left[ \varphi(x(x'),y(y'),z(x',y',r,t),t) \right]
+= \frac{\partial \varphi}{\partial x}\frac{\partial x}{\partial x'} +  \frac{\partial \varphi}{\partial z} \frac{\partial z}{\partial x'}
+$$ (dvarphidx)
+
+We may define the tilted horizontal variable $x'$ as we please. The simplest definition is $x'(x)\equiv x$. Then $\partial x / \partial x'=1$. Rearranging [](#dvarphidx) and repeating for $y$,
+
+$$
+\begin{aligned}
+\frac{\partial \varphi}{\partial x} &=
+\frac{\partial \varphi}{\partial x'}
+- \frac{\partial \varphi}{\partial z} \frac{\partial z}{\partial x'}\\
+\frac{\partial \varphi}{\partial y} &=
+\frac{\partial \varphi}{\partial y'}
+- \frac{\partial \varphi}{\partial z} \frac{\partial z}{\partial y'}.
+\end{aligned}
+$$ (dvarphidxy)
+
+This may be written in vector form as
+
+$$
+\nabla_z \varphi = \nabla_r \varphi - \frac{\partial \varphi}{\partial z} \nabla_r z.
+$$ (dvarphidnabla)
+
+### Pressure Gradient
```

Comment:
I would lean toward leaving out this section entirely.  The balance between the pressure-gradient and gravity terms is fraught enough to be the subject of entire papers and there's a reason we have this in its own design document (which is evolving).  In particular, I don't think I agree with the reasoning behind equation (35) below.

Reply:
I think this section is clear now. You can see the [compiled version here](https://portal.nersc.gov/project/e3sm/mpeterse/omega/design-doc-v1-eqns/_build/html/design/OmegaV1GoverningEqns.html#pressure-gradient). Once I defined the geopotential and substitute everything in, the steps to the final $-v  \nabla_r p - \nabla_r \Phi$ is straightforward.

Reply:
Okay, I think I agree.  Let's discuss it in a few minutes (if we get this far).  I thought the substitution wasn't necessary because the equation already has a geopotential term but I think I see how that term cancels out.